purejin/index.md

```diff
@@ -98,19 +79,19 @@ trouble especially with large, complex applications.
 * use your list/set implementations instead of javas (is really easy)
 * binds different configurations without `if`s everywhere (built-in concepts)
 * binds in different scopes
-* use of your own scopes/lifecycles (is really easy)
-* use of your own provider interface to escape scoping problems (is really easy) 
+* uses your own scopes/lifecycles (is really easy)
+* uses your own provider interface to escape scoping problems (is really easy) 
```

Comment:
This is more confusing now, it does not use _your own_ out of the box but it is easy to setup _your own_.

Reply:
Updated. What do you think?

Reply:
Better, Thanks. I apply it to the scopes item as well.